src/components/GhostCard.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
I think this file moved to some package like `src/components/cards`, `src/components/ghosts`, or etc.
How about you?

Reply:
`src/components/cards` looks better